src/viewport.cpp

```diff
@@ -1726,7 +1791,7 @@ void ViewportDoDraw(const ViewPort *vp, int left, int top, int right, int bottom
  * Make sure we don't draw a too big area at a time.
  * If we do, the sprite memory will overflow.
  */
-static void ViewportDrawChk(const ViewPort *vp, int left, int top, int right, int bottom)
+void ViewportDrawChk(const Viewport *vp, int left, int top, int right, int bottom)
 {
 	if ((int64)ScaleByZoom(bottom - top, vp->zoom) * (int64)ScaleByZoom(right - left, vp->zoom) > (int64)(180000 * ZOOM_LVL_BASE * ZOOM_LVL_BASE)) {
```

Comment:
Where did the magic number *180000* come from? Is it something to be concerned about?

Reply:
There is a significant performance penalty associated with drawing too large an area in one go, because of the poor performance characteristics of the sprite sorter. The comment talks about sprite memory but that is mostly a non-issue.

The exact value of the threshold is not important, it just has to be of roughly the right order of magnitude.

Reply:
Okay, so the exact value is not important. Can we at least find a meaningful name for it?